packages/graphql-testx/package.json

```diff
@@ -7,6 +7,8 @@
     "clean": "rm -rf dist/ node_modules/",
     "build": "tsc",
     "test": "ava",
+    "postinstall": "cd examples/apollo-client && yarn install && cd ../karma && yarn install && cd ../offix && yarn install",
```

Comment:
I do not think that we need postInstall. 
Postinstall is actually wrong as it will be also executed on installed package.
I do not think that we need to run anything else than just unit tests on this package at the moment

Reply:
We actually run unit tests on examples along with the tests on the package itself. I made it so that all the tests that were run by CI before are also run now

Reply:
The unit tests need the dependencies, I will change it so that only running the tests will install them

Reply:
So let's add them to dev dependencies. Any hack on this is not desired. No matter if postinstall or postTest